README.md

```diff
@@ -185,32 +182,272 @@ run
+suggested below.
+
+* provider.google: version = "~> 2.12"
+* provider.random: version = "~> 2.2"
+* provider.template: version = "~> 2.1"
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
+Success! The configuration is valid.
+
+helpers/terraform_validate ./test/fixtures/instance_with_attached_disk
+Initializing modules...
+
+Initializing provider plugins...
+
+The following providers do not have any version constraints in configuration,
+so the latest version was installed.
+
+To prevent automatic upgrades to new major versions that may contain breaking
+changes, it is recommended to add version = "..." constraints to the
+corresponding provider blocks in configuration, with the constraint strings
+suggested below.
+
+* provider.external: version = "~> 1.2"
+* provider.google: version = "~> 2.12"
+* provider.local: version = "~> 1.3"
+* provider.random: version = "~> 2.2"
+* provider.tls: version = "~> 2.0"
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
+Success! The configuration is valid.
+
+helpers/terraform_validate ./test/fixtures/simple_instance
+Initializing modules...
+
+Initializing provider plugins...
+
+The following providers do not have any version constraints in configuration,
+so the latest version was installed.
+
+To prevent automatic upgrades to new major versions that may contain breaking
+changes, it is recommended to add version = "..." constraints to the
+corresponding provider blocks in configuration, with the constraint strings
+suggested below.
+
+* provider.external: version = "~> 1.2"
+* provider.google: version = "~> 2.12"
+* provider.random: version = "~> 2.2"
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
+Success! The configuration is valid.
+
+Checking for required files LICENSE README.md
```

Comment:
I also agree it's too much text for readme

Reply:
I agree,  this is definitely an overkill, but we have an example of successful linters run there. I can't really shrink it much.. This is the way how it looks now. The `terraform validate` and `generate_docs` are way too verbose. 

And then we have `make -s` (silent) or less verbose to very least. 
How do you feel about logging another issue in [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/test/make.sh) or [CFT](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh) repo for making linters silent/less verbose with `-s` ? 
@kopachevsky @ingwarr @paulpalamarchuk @nick4fake

Reply:
Personally, I have nothing against it, because if tests fail, then they fail and in this case, more verbalization is needed, and this will be solved locally during debugging

Reply:
This can be addressed later.

Reply:
> How about we reduce the amount of repetitive text. We don't need to show the entire output of every directory being linted, just a sample of what success looks like.

@aaron-lane, sure. We also discussed making some re-factoring to linters so it can be copy-pasted as entire output and be short as well. I just logged that as [cft 290](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/290)